README.txt

```diff
@@ -214,6 +214,14 @@ Output is tab-delimited to make it easy to paste into a spreadsheet.
 
 === Stubs
 
+Stubs simulate the behaviors of software components (or modules) that a module undergoing tests depends on. Stubs are defined in the following way:
+
+    Klass.stub :method, return_value, block_argument do   # stub goes away once the block is done
```

Comment:
But, stubs aren't class specific. They can be run on any instance, as demonstrated in our previous dialog.